src/App.js

```diff
@@ -1,13 +1,112 @@
 import React from 'react';
+import "./components/TodoComponents/Todo.css"
+import Todo from './components/TodoComponents/Todo';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTask: "",
+      todos: [
```

Comment:
Try to put the data in a data.js file then import it and use setState to add it to your state

Reply:
Understood.